### PR TITLE
FINAP-9/Swedish and English translations

### PR DESCRIPTION
**Pull request by esuomi**
# Added
* Swedish and English translations for Traficom notification box on the front page

### CODE REVIEW

File: ote/resources/public/language/en.edn [hunk 1]
```diff
@@ -393,6 +393,7 @@ You can also draw the operating area or point to the map with drawing tools. You
   :register-to-service "Register"
   :check-out-the-service "Explore the information"
   :continue-editing "Continue modifying the information"
+  :visit-service-operator "Visit Traficomin website"
```
Comment (antti-nuortimo-solita) [inline ote/resources/public/language/en.edn:396]: "Visit Traficom website" I assume.

**Review by antti-nuortimo-solita — COMMENTED** [review]
Typo in English translation "Traficomin" -> "Traficom", and the use of "FINAP" vs. "Finap" seems inconsistent between different translations (in the other PR), otherwise looks good to me :ship:

**Comment by esuomi** [comment]
fix odd Finglish translation

**Comment by esuomi** [comment]
@antti-nuortimo-solita  Copy-paste from customer is what it is :)

**Pull request merged**